icubam/backoffice/handlers/base.py

```diff
@@ -35,7 +35,7 @@ def render(self, path, **kwargs):
 
   def render_list(self, data, objtype, create_handler=None, **kwargs):
     route = None if create_handler is None else create_handler.ROUTE
-    columns = json.dumps([x['key'] for x in data[0]])
+    columns = [] if not data else json.dumps([x['key'] for x in data[0]])
```

Comment:
The output of json.dumps is a string, so we would rather want something like,

```suggestion
    if not data:
        columns = '[]'
    else:
        columns = json.dumps([x['key'] for x in data[0]])
```

to preserve type. 

For tests, we would likely need to add a test in `backoffice/server_test.py` that tries to render the `/list-tokens` endpoint when there are none but I haven't looked in detail. See https://github.com/icubam/icubam/issues/251#issuecomment-612951088 for related discussion.

Reply:
Good catch, thanks! I'm too used to statically typed languages, where you can rely on the compiler :)

Reply:
good catch. I would rather do something like
```
item = data[0] if data else []
columns = json.dumps([x['key'] for x in item])
```